packages/flutter/test/cupertino/popup_surface_test.dart

```diff
@@ -0,0 +1,328 @@
+            blurSigma: 0,
+            isSurfacePainted: false,
+            isVibrancePainted: false,
+            child: SizedBox(),
+          ),
+        ),
+      );
+
+      // The BackdropFilter widget should not be mounted when blurSigma is 0 and
+      // CupertinoPopupSurface.isVibrancePainted is false.
+      expect(
+        find.descendant(
+          of: find.byType(CupertinoPopupSurface),
+          matching: find.byType(BackdropFilter),
+        ),
+        findsNothing,
+      );
+
+      // Golden should display color tiles without CupertinoPopupSurface being
+      // displayed.
+      await expectLater(
+        find.byType(CupertinoApp),
+        matchesGoldenFile('cupertinoPopupSurface.color-filter.removed.png'),
+      );
+    });
+  });
+  group('Surface Color', () {
+    testWidgets('Brightness.light surface', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const CupertinoApp(
+          home: ColoredBox(
+            color: Color(0xff000000),
+            child: Stack(
+              fit: StackFit.expand,
+              children: <Widget>[
+                Padding(
+                  padding: EdgeInsets.all(32),
+                  child: CupertinoPopupSurface(
+                    blurSigma: 0,
+                    isVibrancePainted: false,
+                    child: SizedBox(),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      );
+
+      // Golden displays the surface color of the CupertinoPopupSurface
+      // in light mode.
+      await expectLater(
+        find.byType(CupertinoApp),
+        matchesGoldenFile('cupertinoPopupSurface.surface-color.light.png'),
+      );
+    });
+    testWidgets('Brightness.dark surface', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const CupertinoApp(
+          home: ColoredBox(
+            color: Color(0xffffffff),
+            child: Padding(
+                padding: EdgeInsets.all(32),
+                child: Stack(
+                  fit: StackFit.expand,
+                  children: <Widget>[
+                    CupertinoTheme(
+                      data: CupertinoThemeData(brightness: Brightness.dark),
+                      child: CupertinoPopupSurface(
+                        blurSigma: 0,
+                        isVibrancePainted: false,
+                        child: SizedBox(),
+                      ),
+                    ),
+                  ],
+                )),
+          ),
+        ),
+      );
+
+      // Golden displays the surface color of the CupertinoPopupSurface
+      // in dark mode.
+      await expectLater(
+        find.byType(CupertinoApp),
+        matchesGoldenFile('cupertinoPopupSurface.surface-color.dark.png'),
+      );
+    });
+    testWidgets('Setting isSurfacePainted to false removes the surface color', (WidgetTester tester) async {
+      await tester.pumpWidget(const CupertinoApp(
+        home: ColoredBox(
```

Comment:
```suggestion
      await tester.pumpWidget(
        const CupertinoApp(
          home: ColoredBox(
```

And subsequent intentations.

Reply:
Done!

Reply:
Done! I should note that this was the only one I saw, so let me know if you were thinking of any other place to indent.